CanalTP/Chaos
# Upgrade on channels required

# Description

This PR add some check on channels in order to report an error if a required channel is not send

## Issue

Issue link: BOT-1089

## How to test

Here are the following steps to test this pull request:

- create or update an impact with no messages entry (with a client who have at least one channel with required=true)
- you should have a 400 response from the API with a distinctive message
